crates/holochain/src/core/ribosome/host_fn/remote_signal.rs

```diff
@@ -5,22 +5,20 @@ use holochain_p2p::HolochainP2pCellT;
 use holochain_zome_types::signal::RemoteSignal;
 use holochain_zome_types::zome::FunctionName;
 use holochain_zome_types::zome::ZomeName;
-use holochain_zome_types::RemoteSignalInput;
-use holochain_zome_types::RemoteSignalOutput;
 use std::sync::Arc;
 use tracing::Instrument;
 
 #[tracing::instrument(skip(_ribosome, call_context, input))]
 pub fn remote_signal(
     _ribosome: Arc<impl RibosomeT>,
     call_context: Arc<CallContext>,
-    input: RemoteSignalInput,
-) -> RibosomeResult<RemoteSignalOutput> {
+    input: RemoteSignal,
+) -> RibosomeResult<()> {
```

Comment:
see above

Reply:
there's no response from a signal, it's a best-effort push only and we don't wait for a response